### PR TITLE
Closes #2490: Fixed confusing string in site permissions prompt [ci skip].

### LINKED ISSUE #2490: Site permissions: Confusing string mozac_feature_sitepermissions_selfie_camera
The sitepermissions string says "back camera" but the description in the comment above says "font facing".

```
    <!-- Option in a dialog for requesting a camera permission, this option will give access to
     front facing camera-->
    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie back camera</string>
```

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/feature/sitepermissions/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
      front facing camera-->
-    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie back camera</string>
+    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie camera</string>
```
Comment (pocmo) [inline components/feature/sitepermissions/src/main/res/values/strings.xml]: @Pike @Delphine @flodolo What's your recommendation here? Should we update the string id when doing such a change?

Comment (Pike) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: I wish this was a no, but apparently it's a yes, https://pontoon.mozilla.org/de/android-l10n/all-resources/?search=camera&string=196924.

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: @pocmo should I update the string id? :(

Comment (pocmo) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: Yes, let's do it. :)

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: 👍

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: @pocmo  I'm renaming the `mozac_feature_sitepermissions_selfie_camera` to    `mozac_feature_sitepermissions_front_facing_camera`. Should I delete the old strings?

image

**Comment by mheubusch** [comment]
Hi Arturo - I just redelivered strings yesterday to pocmo as an xml file
and used front-facing and rear-facing camera for the options.  i'm find to
use selfie and rear-facing camera and want to make sure i haven't created a
conflict in the file.

**Comment by Amejia481** [comment]
Closing in favor of https://github.com/mozilla-mobile/android-components/pull/2508

**Pull request closed without merging**